internal/runners/packages/import.go

```diff
@@ -144,12 +135,51 @@ func (i *Import) Run(params *ImportRunParams) error {
 		return locale.WrapError(err, "err_commit_changeset", "Could not commit import changes")
 	}
 
-	if err := localcommit.Set(i.proj.Dir(), commitID.String()); err != nil {
+	pg.Stop(locale.T("progress_success"))
+	pg = nil
+
+	// Solve the runtime.
+	rt, rtCommit, err := runtime.Solve(auth, out, i.prime.Analytics(), proj, &commitID, target.TriggerImport, i.prime.SvcModel(), i.prime.Config(), runtime.OptNone)
+	if err != nil {
+		return errs.Wrap(err, "Could not solve runtime")
+	}
```

Comment:
This made me realize we're redundantly solving. Not asking for this to be addressed here, but for future context: https://activestatef.atlassian.net/browse/DX-2934

Filed it separately as this could easily get more complicated than just updating the model code and using the result from stageCommit directly (ie. UI implications).